self-core-impl/src/main/java/com/selfxdsd/core/GitlabCollaborators.java

```diff
@@ -125,4 +132,27 @@ public boolean remove(final String username) {
         return result;
     }
 
+    @Override
+    public Iterator<Collaborator> iterator() {
+        LOG.debug(
+            "Fetching Gitlab's repo collaborators "
+                + "from [" + this.collaboratorsUri.toString() + "]."
+        );
+        final Resource response = this.resources.get(this.collaboratorsUri);
+        final List<Collaborator> collaborators;
+        if (response.statusCode() == HttpURLConnection.HTTP_OK) {
+            collaborators = response.asJsonArray()
+                .stream()
+                .map(JsonObject.class::cast)
+                .map(GitlabCollaborator::new)
+                .collect(Collectors.toList());
+        } else {
+            throw new IllegalStateException(
+                "Unable to fetch Gitlab collaborators"
+                    + " from [" + this.collaboratorsUri.toString() + "],"
+                    + " 200 was expected but we got " + response.statusCode()
+            );
```

Comment:
@fellahi-ali Let's not throw an exception because Collaborators iteration will not be used in any critical flow.

If the response is != HTTP_OK, let's just log an error and assign an empty list.

Reply:
@amihaiemil done, thanks :)